deploy/olm-catalog/infinispan-operator.clusterserviceversion.yaml

```diff
@@ -80,33 +80,145 @@ spec:
+            path: service.sites.locations[].secretName
+            x-descriptors:
+              - 'urn:alm:descriptor:io.kubernetes:Secret'
+          - displayName: Node port
+            description: Node port to which the load balancer service forwards traffic
+            path: service.sites.local.expose.nodePort
+            x-descriptors:
+              - 'urn:alm:descriptor:com.tectonic.ui:number'
+              - 'urn:alm:descriptor:com.tectonic.ui:fieldDependency:service.sites.local.expose.type:NodePort'
+          - displayName: Cache replication factor
+            description: Configures replicas for each cache entry
+            path: service.replicationFactor
+            x-descriptors:
+              - 'urn:alm:descriptor:com.tectonic.ui:number'
+          - description: Enable/disable container ephemeral storage
+            displayName: Container ephemeral storage
+            path: service.container.ephemeralStorage
+            x-descriptors:
+              - 'urn:alm:descriptor:com.tectonic.ui:booleanSwitch'
+          - displayName: Storage size
+            description: The size of the storage to use in the persistent volume claim
+            path: service.container.storage
+            x-descriptors:
+              - 'urn:alm:descriptor:text'
+          - displayName: Storage class name
+            description: The name of the storage class object to use for the persistent volume claim
+            path: service.container.storageClassName
+            x-descriptors:
+              - 'urn:alm:descriptor:io.kubernetes:StorageClass'
+              - 'urn:alm:descriptor:com.tectonic.ui:fieldDependency:service.container.ephemeralStorage:false'
+          - displayName: Expose host name for the Route
+            description: Specifies the hostname where Infinispan is exposed
+            path: expose.host
+            x-descriptors:
+              - 'urn:alm:descriptor:com.tectonic.ui:text'
+              - 'urn:alm:descriptor:com.tectonic.ui:fieldDependency:expose.type:Route'
+          - displayName: Persistent volume claim name
+            description: Name of the persistent volume claim with custom libraries
+            path: dependencies.volumeClaimName
+            x-descriptors:
+              - 'urn:alm:descriptor:io.kubernetes:PersistentVolumeClaim'
         statusDescriptors:
           - description: The current pods
             displayName: Pods Status
             path: podStatus
             x-descriptors:
-              - "urn:alm:descriptor:com.tectonic.ui:podStatuses"
-      - description: Restore is the Schema for the restores API
-        kind: Restore
-        name: restores.infinispan.org
-        version: v2alpha1
-        displayName: Restore
+              - 'urn:alm:descriptor:com.tectonic.ui:podStatuses'
   description: |
       Infinispan is a distributed, in-memory data store that increases application performance and delivers open-source capabilities to handle demanding use cases.
```

Comment:
```suggestion
      Infinispan is a distributed in-memory data store built on open-source technology.
```

Reply:
Could you please review whole description, but I'm not owner and author if it

Reply:
Thanks @dmvolod I remember writing that back in the early days of the operator. I kind of feel like we can trim it down but maybe it should stay the same to match what's in the CSV for the bundle: http://pkgs.devel.redhat.com/cgit/containers/datagrid-8-prod-operator-bundle/tree/manifests/datagrid.clusterserviceversion.yaml?h=datagrid-8.1-rhel-8#n120 

@rigazilla and @ryanemerson do either of you have an opinion on this description?

Reply:
@oraNod, feel free to improve the description, imo staying close to RHDG 8.1 is not a requirement for master branch.